Dockerfile

```diff
@@ -23,7 +23,12 @@ ENV NODE_OPTIONS="--max-old-space-size=2048"
 RUN cd /srv/CyberChef && \
     grunt prod
 
+WORKDIR /srv/CyberChef/build/prod
+
+RUN REL=$(curl -ksL "https://api.github.com/repos/gchq/CyberChef/releases/latest" | jq -r '.tag_name') && \
+    curl -s -L -O "https://github.com/gchq/CyberChef/releases/download/${REL}/CyberChef_${REL}.zip" && \
+    unzip -u CyberChef_${REL}.zip
+
```

Comment:
I am working on an update to this image and I have a question to these lines. Why do you build cyberchef and then download the pre-build release?

Reply:
@rxcket indeed that looks a bit nasty. I think the answer is in #3. Since this was an issue over two years ago, I do think we can have a poke at cleaning this up.